dashboard/api.py

```diff
@@ -1 +1,92 @@
 import requests
+
+def get_visit_events(project_name, after_time, limit, after_id=None)
+    url = "analytics.bitsofgood.org/api/events/visit-events"
+    params = {
+        "projectName": project_name,
+        "afterTime": after_time,
+        "limit": limit,
+    }
+    if after_id:
+        params["afterId"] = after_id
+
+    try:
+        response = requests.get(url, params=params)
+        response.raise_for_status()  # error
+        return response.json()
+    except requests.exceptions.RequestException as err:
+        return {"error": "Request failed", "details": str(err)}
```

Comment:
lets keep the return json consistent so instead of returning this, return something like

```
{
"success" : false,
"payload": {
"error": str(err)
}
}
```